lib/internal/errors.js

```diff
@@ -1308,8 +1308,12 @@ E('ERR_INVALID_CURSOR_POS',
 E('ERR_INVALID_FD',
   '"fd" must be a positive integer: %s', RangeError);
 E('ERR_INVALID_FD_TYPE', 'Unsupported fd type: %s', TypeError);
-E('ERR_INVALID_FILE_URL_HOST',
-  'File URL host must be "localhost" or empty on %s', TypeError);
+E('ERR_INVALID_FILE_URL_HOST', (url, platform, base) => {
+  let msg = `File URL host in ${url}`;
```

Comment:
Why not pass the base and url to this just like in err_invalid_url?

Reply:
Looking at the comment within `err_invalid_url` that's probably a good idea!